spec/unit_tests/moab/storage_object_validator_spec.rb

```diff
@@ -15,104 +15,137 @@
+        storage_obj = Moab::StorageObject.new(druid, druid_path)
+        storage_obj_validator = described_class.new(storage_obj)
+        verification_array = storage_obj_validator.validation_errors
+        expect(verification_array).to include(Moab::StorageObjectValidator::INCORRECT_DIR => "Incorrect items in path")
       end
-      it 'has missing content directory' do
-        # v0006
-        expect(verification_array[6]).to eq(Moab::StorageObjectValidator::MISSING_DIR =>"Missing directory: [\"content\", \"metadata\"] Version: v0006")
+      it "has incorrect version directory name" do
+        druid = 'zz000zz0000'
+        druid_path = 'spec/fixtures/bad_root01/bad_moab_storage_trunk/zz/000/zz/0000/zz000zz0000'
+        storage_obj = Moab::StorageObject.new(druid, druid_path)
+        storage_obj_validator = described_class.new(storage_obj)
+        verification_array = storage_obj_validator.validation_errors
+        expect(verification_array).to include(Moab::StorageObjectValidator::VERSION_DIR_BAD_FORMAT =>"Version directory name not in 'v00xx' format")
       end
-      it 'has no required items' do
-        # v0006
-        expect(verification_array[7]).to eq(Moab::StorageObjectValidator::NO_XML_FILES =>"Version: v0006 Missing all required metadata files")
+      it 'does not call #check_expected_data_sub_dirs because moab does not have the expected version sub dirs' do
+        druid = 'ee000ee0000'
+        druid_path = 'spec/fixtures/bad_root01/bad_moab_storage_trunk/ee/000/ee/0000/ee000ee0000'
+        storage_obj = Moab::StorageObject.new(druid, druid_path)
+        storage_obj_validator = described_class.new(storage_obj)
+        expect(storage_obj_validator).not_to receive(:check_expected_data_sub_dirs)
+        storage_obj_validator.validation_errors
       end
-    end
-    context 'under manifest directory' do
-      druid = 'cc000cc0000'
-      druid_path = 'spec/fixtures/bad_root01/bad_moab_storage_trunk/cc/000/cc/0000/cc000cc0000'
-      storage_obj = Moab::StorageObject.new(druid, druid_path)
-      storage_obj_validator = described_class.new(storage_obj)
-      verification_array = storage_obj_validator.validation_errors
-      it 'does not have signatureCatalog.xml' do
-        expect(verification_array).to include(Moab::StorageObjectValidator::NO_SIGNATURE_CATALOG => "Version: v0001 Missing signatureCatalog.xml")
+      it 'does not call #check_metadata_dir_file_only because moab does not have the expected data sub dirs' do
+        druid = 'ff000ff0000'
+        druid_path = 'spec/fixtures/bad_root01/bad_moab_storage_trunk/ff/000/ff/0000/ff000ff0000'
+        storage_obj = Moab::StorageObject.new(druid, druid_path)
+        storage_obj_validator = described_class.new(storage_obj)
+        expect(storage_obj_validator).not_to receive(:check_metadata_dir_file_only)
+        storage_obj_validator.validation_errors
       end
```

Comment:
do we now need tests to show what happens when manifest dir passes, vs. when manifest dir fails?

Reply:
manifest dir fails in line 74,78 (does not have _required_ files - sigCat.xml and manInv.xml)

Reply:
i'll add a test for when manifest dir passes though